stripe/src/main/java/com/stripe/android/model/SourceParams.java

```diff
@@ -37,8 +45,13 @@
     static final String FIELD_STATE = "state";
     static final String FIELD_STATEMENT_DESCRIPTOR = "statement_descriptor";
 
+    static final Set<String> NO_AMOUNT_SOURCES = new HashSet<String>() {{
+        add(Source.CARD);
+        add(Source.SEPA_DEBIT);
+    }};
+
     @IntRange(from = 0) private long mAmount;
-    private Map<String, Map<String, Object>> mApiParameterMap;
```

Comment:
Why did I ever make this a two-layer map? I think I was counting on auto-stringifying it later, but doing it as a simple map is way easier.